src/client/testing/page-objects/change-site-details-pages/site-details.ts

```diff
@@ -18,20 +19,38 @@ export default class SiteDetailsPage extends RootPage {
   readonly latitudeLabel = 'Latitude';
   readonly longitudeLabel = 'Longitude';
   readonly phoneNumberLabel = 'Phone Number';
-
-  readonly defaultSiteName = 'Church Lane Pharmacy';
-  readonly defaultAddress = 'Pudsey, Leeds, LS28 7LD';
-  readonly defaultLatitude = '-1.66382134';
-  readonly defaultLongitude = '53.79628754';
-  readonly defaultPhoneNumber = '0113 2222222';
-
   readonly odsCodeLabel = 'ODS code';
   readonly icbLabel = 'ICB';
   readonly regionLabel = 'Region';
 
-  readonly defaultODSCode = 'ABC02';
-  readonly defaultICB = '	Integrated Care Board 2';
-  readonly defaultRegion = 'Region 2';
+  readonly sites: Site[] = [
+    {
+      id: 'test-1',
+      address: 'Pudsey, Leeds, LS28 7LD',
+      name: 'Church Lane Pharmacy',
+      location: {
+        coordinates: [-1.66382134, 53.79628754],
+        type: 'point',
+      },
+      phoneNumber: '0113 2222222',
+      odsCode: 'ABC02',
+      integratedCareBoard: 'Integrated Care Board 2',
+      region: 'Region 2',
+    },
+    {
+      id: 'test-2',
+      address: 'Pudsey, Leeds, LS28 7BR',
+      name: 'Robin Lane Medical Centre',
+      location: {
+        coordinates: [-1.6610648, 53.795467],
+        type: 'point',
+      },
+      phoneNumber: '0113 1111111',
+      odsCode: 'ABC01',
+      integratedCareBoard: 'Integrated Care Board 1',
+      region: 'Region 1',
+    },
+  ];
```

Comment:
We shouldn't be putting hardcoded seed data in our page objects. This will end up getting duplicated the next time another page object needs it or becoming really hard to maintain and increasing the chance of manual errors. 

All data like this should be imported from an external source. We could put it into the config, or create a separate seed data folder within the playwright structure. 

We then only populate the page object with seed data at runtime, either by passing the data in as arguments to methods or as a property within the page object constructor.